Misc/NEWS.d/next/Library/2018-06-26-19-03-56.bpo-33871.XhlrGU.rst

```diff
@@ -0,0 +1,4 @@
+Fixed integer overflow in :func:`os.readv`, :func:`os.writev`,
+:func:`os.preadv` and :func:`os.pwritev` and in :func:`os.sendfile` with
+*headers* or *trailers* arguments (on BSD-based OSes and MacOS).
+Thanks Ned Deily for testing on 32-bit MacOS.
```

Comment:
MacOS -> macOS.  Also, thanks for the "thanks" but that is not needed for a NEWS entry.